src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -1025,6 +1049,28 @@ UniValue AsyncRPCOperation_sendmany::perform_joinsplit(
         arrOutputMap.push_back(outputMap[i]);
     }
 
+
```

Comment:
This code is very similar to what happens inside of `shieldcoinbase` and so should be merged together into a common abstraction.

Reply:
Yes, will refactor later.